packages/flutter_tools/test/commands.shard/hermetic/pub_test.dart

```diff
@@ -16,6 +16,17 @@ import 'package:test/fake.dart';
 import '../../src/context.dart';
 import '../../src/test_flutter_command_runner.dart';
 
+const String minimalV2EmbeddingManifest = r'''
```

Comment:
I added this to make the these pub get tests report v2 embedding, instead of v1, because the tests themselves were largely unrelated to testing android embedding behavior and thus it didn't make sense to delete them.

But I don't really know why running `pub get` should be at all linked to the android embedding version? I don't know why we compute and return the [android embedding version here](https://github.com/flutter/flutter/blob/46cb2d7b013cd356962ed41d55fba2e65ddb89f2/packages/flutter_tools/lib/src/commands/packages.dart#L429).

Reply:
Seems maybe analytics related, in which case it would also be a good candidate for removal (Not the test, but the behavior of linking the android embedding version to pub get here). We don't need analytics for deleted code.

Reply:
It does indeed look like this was analytics related: https://github.com/flutter/flutter/pull/44043

Reply:
I removed what I could of the analytics. Some pieces of it, [like this `Event` class](https://github.com/flutter/flutter/blob/46cb2d7b013cd356962ed41d55fba2e65ddb89f2/packages/flutter_tools/lib/src/commands/packages.dart#L460), are not part of the flutter repo? I don't know where they are defined or what the process for removing the embedding version from that class would be like.

Reply:
Noting after merging for anyone who comes across this pr in search: I decided to not delete the analytics, as we will probably have a v3 embedding one day